services/QuillLMS/app/queries/quill_big_query.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+# All messages to BigQuery should use this module
+
+require 'google/api_client'
+require 'googleauth'
+require 'stringio'
+
+module QuillBigQuery
+  PROJECT_ID = 'analytics-data-stores'
+  FLOAT_FIELDS = %w(correct incorrect percentage)
```

Comment:
Hmm, this `FLOAT_FIELDS` constant only applies to this query. I don't think this is going to scale well, e.g. every query you'd have to remember to update this constant, plus what if there is an overlap in field names among different queries (correct is a float here, but could be a boolean in a different query). 

There is a field schema returned by API. I think we'd want to use that to determine any transforms. That's a more scalable approach.